app/src/main/assets/ime/keyboard/org.florisboard.layouts/extension.json

```
@@ -17,6 +17,20 @@
         "direction": "rtl",
         "modifier": "org.florisboard.layouts:arabic"
       },
+      {
+        "id": "western_armenian",
+        "label": "Armenian",
```

Naming two different layouts with the same label will confuse the user. Modifying the label to clarify helps:

```suggestion
        "label": "Armenian (Western)",
```